telemetry/data.go

```diff
@@ -43,11 +44,16 @@ func generateTelemetryData(ctx sessionctx.Context, trackingID string) telemetryD
 	if f, err := getFeatureUsage(ctx); err == nil {
 		r.FeatureUsage = f
 	}
+	if s, err := getSlowQueryStats(ctx); err == nil {
+		r.SlowQueryStats = s
+	}
```

Comment:
do we need log error in here, it seems some error didn't have be logged in under method

for example: errors.New("Prom vector expected, got " + value.Type().String())

Reply:
All the errors in getSlowQueryStats have been logged, we needn't do it again here.
func getSlowQueryStats(ctx sessionctx.Context) (*slowQueryStats, error) {
	slowQueryBucket, err := getSlowQueryBucket(ctx)
	if err != nil {
		logutil.BgLogger().Info(err.Error())
		return nil, err
	}

Reply:
https://github.com/pingcap/tidb/blob/b21aed8cc46e229b74fcd4e96e9f4170ceecde98/telemetry/data_slow_query.go#L89

it seems this error have no chance be logged